README.adoc

```diff
@@ -103,6 +103,12 @@ Do NOT use it.
 |true
 |false
 
+.2+|dnsjava.hostsfile.max_size
```

Comment:
Please name the property `dnsjava.hostsfile.max_size_bytes`, otherwise this might be confused with the number of lines to read.

Reply:
The property name is changed now, but not the doc.